### DISCUSSION TITLE
Using Pennant for tiered SaaS features

### DISCUSSION THREAD

**Discussion by sts-ryan-holton** [laravel/framework#51387] [Q&A]
Hi, I'm wondering whether Laravel Pennant is the right choice for a SaaS application that defines various "tiered" pricing options where each option has limits on it's features, for example, I might have 3 plans, a free plan, team and business. On each plan I might have various features that define the limits for the plan, for example, the free plan might be restricted to 5 seats, 10 on the team and 20 on the business.

How would I go about using pennant to create and define these various limits?

**Comment by a-h-abid** [comment]
Pennant is feature toggles for unreleased, incomplete developed features. It is not made for Saas options toggles.

You have to use some other library or make one your own.
